src/colab/Run_SKAI_XManager_Colab_Pipeline.ipynb

```diff
@@ -69,17 +69,17 @@
         "#@title Please run this cell first!\n",
         "\n",
         "#@markdown Specify the parameters to set up your Colab notebook. They should be the same that the ones used during the initialization of the Colab kernel\n",
-        "#############################################\n",
-        "### CODE SETTING - ENVIRONMENT ACTIVATION ###\n",
-        "#############################################\n",
+        "###################################################################################\n",
+        "### CODE SETTING - ENVIRONMENT ACTIVATION and Installation of required packages ###\n",
```

Comment:
If not need for the virtual environment, please remove any reference to it in the notebook

Reply:
ok